package.json

```diff
@@ -14,7 +14,7 @@
     "deploy": "sh scripts/deploy.sh"
   },
   "dependencies": {
-    "@artsy/reaction-force": "^0.1.49",
+    "@artsy/reaction-force": "^0.3.1",
```

Comment:
Not sure if you want to include any of the latest Reaction work here, but I imagine we should be at a 0.4.0 now if you want it.